xbmc/music/windows/GUIWindowMusicNav.cpp

```diff
@@ -287,6 +287,10 @@ bool CGUIWindowMusicNav::GetDirectory(const std::string &strDirectory, CFileItem
       OnRetrieveMusicInfo(items);
   }
 
+  //Navigating music files so default content to "songs" unless in sources folder.
+  //This content allows view type to include media info so that file tag data can be displayed
+  if (!URIUtils::IsSourcesPath(strDirectory))
```

Comment:
> Have you ever been to the video files views? It also shows all the library details for all video files that have been scanned into the library. So it's not like the video files view is just a list of files in their raw state.

This could be a bit heavy for music collections if done the same way as we're handling it for videos.